engine/orchestration/test/com/cloud/vm/VirtualMachineManagerImplTest.java

```diff
@@ -505,4 +513,24 @@ public void testSendStopWithNullAnswer() throws Exception {
 
         Assert.assertFalse(actual);
     }
+
```

Comment:
@swill I've fixed the merge conflicts and merged 4.7 on 4.8; 4.8 on master branches respectively. I've tested master branch by doing a full noredist build.